File: src/main/java/me/captainpotatoaim/myplugin/duct_tape/DuctTape.java
package me.captainpotatoaim.myplugin.duct_tape;

import org.bukkit.command.Command;
import org.bukkit.command.CommandExecutor;
import org.bukkit.command.CommandSender;
import org.bukkit.entity.Player;
import org.jetbrains.annotations.NotNull;

import java.util.HashSet;
import java.util.UUID;

public class DuctTape implements CommandExecutor {
    static HashSet<UUID> tapedPlayers = new HashSet<>();

    public boolean onCommand(@NotNull CommandSender sender, @NotNull Command command, @NotNull String label, @NotNull String[] args) {
        tapedPlayers.add(sender.getServer().getPlayer(args[0]).getUniqueId());

        return true;
    }
}

File: src/main/java/me/captainpotatoaim/myplugin/duct_tape/DuctTape.kt
package me.captainpotatoaim.myplugin.duct_tape

import org.bukkit.command.Command
import org.bukkit.command.CommandExecutor
import org.bukkit.command.CommandSender
import java.util.HashSet
import java.util.UUID

class DuctTape : CommandExecutor {

    override fun onCommand(sender: CommandSender, command: Command, label: String, args: Array<out String>?): Boolean {
        tapedPlayers.add(sender.server.getPlayer(args!![0])?.uniqueId)

        return true
    }

    companion object {
        @JvmField
        var tapedPlayers: HashSet<UUID?> = HashSet<UUID?>()
    }
}
